### ISSUE TITLE
Wrong information in GPG signature verification failed

### ISSUE THREAD

**Issue by torrocus** [rvm/rvm#4553]
## Description

RVM can not be updated
It's probably related with closed issue #4457, but not solved correctly.

## Expected behavior

Successful RVM upgrade.

## Actual behavior

```
GPG signature verification failed for '/home/torrocus/.rvm/archives/rvm-installer' - 'https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer.asc'! Try to install GPG v2 and then fetch the public key:

    gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3

or if it fails:

    command curl -sSL https://rvm.io/mpapis.asc | gpg2 --import -

the key can be compared with:

    https://rvm.io/mpapis.asc
    https://keybase.io/mpapis
```

## Environment info

```
torrocus@knuth ~ $ uname -a
Linux knuth 3.13.0-164-generic #214-Ubuntu SMP Wed Dec 5 10:42:33 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
torrocus@knuth ~ $ gpg2 --version
gpg (GnuPG) 2.0.22
libgcrypt 1.5.3
```

**Comment by pkuczynski** [comment]
This will be fixed in the next release (very soon). Already in master.

And your fix in #4554 is not correct I am afraid.

**Comment by torrocus** [comment]
@pkuczynski  Ok, thanks for info

**Comment by pkuczynski** [comment]
We just released a new version 1.29.7 fixing this issue.

[Become a backer](https://opencollective.com/rvm#backer) and support us with a small donation to help us continue our activities. Thank you if you already one! 🙏
